docs/components/clerk-provider.mdx

```diff
@@ -90,25 +90,24 @@ The `<ClerkProvider>` component must be added to your React entrypoint.
+| `signUpFallbackRedirectUrl?` | `string`                                                                              | The fallback URL to redirect to after the user signs up, if there's no `redirect_url` in the path already. Defaults to `/`. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects) instead.                                                                                                                                                                            |
 | `isSatellite?`            | `boolean \| ((url: URL) => boolean)`                                                  | This option defines that the application is a satellite application.                                                                                                                                                       |
 | `domain?`                 | `string \| ((url: URL) => boolean)`                                                   | This option sets the domain of the satellite application. If your application is a satellite application, this option is required.                                                                                         |
 | `signInUrl`               | `string`                                                                              | This URL will be used for any redirects that might happen and needs to point to your primary application. This option is optional for production instances and required for development instances. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects) instead.                        |
-
-| `telemetry?`               | `false \| { disabled?: boolean; debug?: boolean } \| undefined`                       | Controls whether or not Clerk will collect [telemetry data](/docs/telemetry).                                                                                                                                              |
+| `signUpUrl`               | `string`                                                                              | This URL will be used for any redirects that might happen and needs to point to your primary application. This option is optional for production instances and required for development instances. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects) instead.                        |

```

Comment:
As part of https://github.com/clerk/javascript/pull/3525 - We could go ahead and update `signUpUrl` & `signInUrl` here to be explicit that those are only used by client components.

Reply:
I'll do that in a next PR to bundle all docs changes as part of https://github.com/clerk/javascript/pull/3525